### ISSUE TITLE
run the code on coco, but can not get the same results shown in the paper

### ISSUE THREAD

**Issue by rock4you** [dddzg/up-detr#8]
<img width="594" alt="updetr" src="https://user-images.githubusercontent.com/2437212/125018130-1451d680-e0a7-11eb-81be-2652a4860d92.png">

**Comment by dddzg** [comment]
May I ask you for the detail config of training coco (number of gpus and etc)?
There is a log in our experiments:https://drive.google.com/file/d/1DQqveOZnMc2VaBhMzl9VilMxdeniiWXo/view?usp=sharing
You can compare your log with it.

**Comment by rock4you** [comment]
GPU using 8 cards of V100 , and the commands are the same as your provided in the github.
Is there anything has to be modified before running the train program?

**Comment by dddzg** [comment]
I check it again. There is a mistake of my script. The `lr_backbone` should set to `5e-5` instead of `5e-4`. I will update the README. Thanks a lot! I will keep the issue open until you get the right result.

**Comment by dddzg** [comment]
Hi @rock4you , may I ask for some new progress?

**Comment by rock4you** [comment]
Still running, it looks good this time.
<img width="401" alt="training" src="https://user-images.githubusercontent.com/2437212/125554893-d5057c22-8713-46ba-af65-7c2371a719a3.png">

**Comment by rock4you** [comment]
The AP of coco val2017 with 300 epochs in Table 2 of the paper is 42.8,
is this result get from a certain training process or the mean value of several times ?

**Comment by rock4you** [comment]
Still running, the AP around epoch 240 is 0.430.
The training speed is about 40 epochs / day

**Comment by dddzg** [comment]
Glad to hear the result. As far as I observe, the open-source pre-trained model is a little better than paper report.

**Comment by rock4you** [comment]
👍🏻 👍🏻

**Comment by rock4you** [comment]
0.435

![updetr](https://user-images.githubusercontent.com/2437212/126096285-08aef8dd-b0c6-48f1-84de-9b20fe7ece97.jpeg)

**Comment by dddzg** [comment]
Nice to hear the result. Could you attach more detailed COCO style evaluation result (such like https://gist.github.com/dddzg/cd0957c5643f5656f6cdc979da4d6db1)?

**Comment by rock4you** [comment]
The last epoch:

IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.432
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.632
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.458
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.209
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.475
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.622
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.342
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.550
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.589
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.311
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.650
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.816
Training time 7 days, 12:19:43
-------------------------------------------------------------------------------------
The highest at epoch 288:
IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.435
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.633
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.464
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.211
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.477
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.626
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.342
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.550
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.589
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.315
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.653
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.818

**Comment by xiaoerlaigeid** [comment]
> I check it again. There is a mistake of my script. I am so sorry. The `lr_backbone` should set to `5e-5` instead of `5e-4`. I will update the README. Thanks a lot! I will keep the issue open until you get the right result.

Hi you mean in the finetune stage or pretrain stage ? Why in the pretrain stage the backbone should freeze ?
